### PR TITLE
Post-release

### PR DESCRIPTION
**Pull request by josephignace**
Added post-release scripts

### CODE REVIEW

File: analysis/post_release/manuscript_tableAER.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+df <- df[df$aer_age=="overall" &
+           df$aer_sex=="overall" &
+           df$analysis=="main" & 
+           df$days==196,]
+
+# Add plot labels --------------------------------------------------------------
+print("Add plot labels")
+
+plot_labels <- readr::read_csv("lib/plot_labels.csv",
+                               show_col_types = FALSE)
```
Comment (marwaalarab) [inline analysis/post_release/manuscript_tableAER.R:23]: I can't see the file in lib, is it there ?

Comment (josephignace) [reply analysis/post_release/manuscript_tableAER.R:23]: No, I just added the scripts.

Comment (josephignace) [reply analysis/post_release/manuscript_tableAER.R:23]: Lib file added.

**Review by venexia — APPROVED** [review]

File: analysis/post_release/specify_paths.R
Comment (venexia) [inline analysis/post_release/specify_paths.R:1]: This file should be hidden and an example file without the identifiable information left in its place.

**Review by venexia — CHANGES_REQUESTED** [review]

File: lib/plot_labels.csv [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿term,label
+alzheimer_disease,Alzheimer disease
+any_dementia,Any dementia
+cognitive_impairment,Cognitive impairment
+migraine,Migraine
+motor_neurone_disease,Motor neurone disease
+multiple_sclerosis,Multiple sclerosis
+other_dementias,Other dementias
+parkinson_disease,Parkinson disease
```
Comment (venexia) [inline lib/plot_labels.csv:9]: Please make the label 'Parkinson's disease'

**Review by venexia — APPROVED** [review]

**Pull request merged**